.expeditor/config.yml

```diff
@@ -0,0 +1,21 @@
+# Documentation available at https://expeditor-docs.es.chef.io/
+
+# The name of the product keys for this product (from mixlib-install)
+product_key:
+  - harmony
+  - chef-server
```

Comment:
This isn't quite accurate. I don't think we want to trigger chef-server builds via this repository.